examples/multi_format_example.html

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
```

Comment:
I'm trying to run this test page (with a real hostname instead of 127.0.0.1) and am not seeing any ads rendering.  Is this working?

Reply:
This seems to be fixed now.  I had added the `hb_format` key, but had not deleted the (now-defunct) `hb_mediatype` key, and apparently the keys get ANDed together (?).  At any rate it should work now.